en/setup/fast-rtps-installation.md

```diff
@@ -48,6 +49,9 @@ Clone the project from Github:
 $ git clone https://github.com/eProsima/Fast-RTPS
 $ mkdir Fast-RTPS/build && cd Fast-RTPS/build
 ```
+
+> **Note** You may need to [install Gradle](https://gradle.org/install/) to build the source (e.g. this is true on vanilla Fedora Linux). A build warning will be displayed if this is the case.
```

Comment:
FYI, this is to address the comment: https://github.com/PX4/Devguide/pull/241#issuecomment-320883284